homeassistant/components/broadlink/climate.py

```diff
@@ -50,6 +50,7 @@ def __init__(self, device: BroadlinkDevice) -> None:
         super().__init__(device)
         self._attr_unique_id = device.unique_id
         self._attr_hvac_mode = None
+        self.sensor = 0
```

Comment:
```suggestion
        self.use_external_sensor = False
```

I think the new variable name makes it easier to understand what exactly it is doing. Since in Python3 False is a keyword for 0 and True for 1 I suggest to use a Boolean here to make it clear that only two values are allowed.

Reply:
As you can see below the sensor value is not a boolean, the value can be 2.

Reply:
True. Let me create a new suggestion.

Reply:
I think the new SelectEntity or SensorEntity should be introduced in a separate pull request.